platform/ui/src/components/Tooltip/Tooltip.tsx

```diff
@@ -35,26 +36,46 @@ const Tooltip = ({
   isSticky,
   position,
   className,
+  tooltipBoxClassName,
   tight,
   children,
   isDisabled,
+  showHideDelay,
```

Comment:
can you please document what is showHideDelay and how people can use it?

Reply:
Done